pages/index.tsx

```diff
@@ -7,9 +7,21 @@ import { requestRequiredPermissions, dAppClient } from "../config/wallet";
 import { useEffect } from "react";
 import { RequestSignPayloadInput, SigningType } from "@airgap/beacon-sdk";
 import { payloadBytesFromString } from "../lib/payload";
+import { connectToDatabase } from "@/config/mongo";
+// import { connect } from "@/config/mongo";
 
```

Comment:
Clear all active accounts when we go to the home page. Otherwise, it was automatically login in the previous user.